### ISSUE TITLE
getOembed api v1.1 throws exception; directQuery api v1.1 throws exception

### ISSUE THREAD

**Issue by hexabot** [atymic/twitter#342]
Hi there,

Great work btw!

**Reported Issue:**
I am unable to request embedded tweets. There is an additional full stop in the api url when requesting getOembed call.

Here is the error which is thrown from Client:

**NotFoundException: A request error occurred. Client error: `GET https://api.twitter.com/1.1/oembed.?url=https%3A%2F%2Ftwitter.com%2Felonmusk%2Fstatus%2F[REMOVED]**

PHP-FPM 7.4
Laraval ^8.12
atymic/twitter 3.0
twitter api v1.1

**Attempted workaround:**
In version 3.0 I had to write a directQuery call, since it had stopped working from the version 2.x migration.

I don't yet understand how you interpret which format to return the response as, or why this exception is occurring, so I am also posting this here.

The exception thrown for my directQuery is TwitterClientException, I believe it is caused by SyncClient > request, as formatResponse returns an appropriate array object.

Here is laravels reported error:
**ClientException: cURL error 6: Could not resolve host: GET (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for GET?url=https%3A%2F%2Ftwitter.com%2Felonmusk%2Fstatus%2F[REMOVED]**

PHP-FPM 7.4
Laraval ^8.12
atymic/twitter 3.1
twitter api v1.1

Prior to updating atymic/twitter to version 3.1 earlier today, getOembed fails, and my workaround succeeded. Post upgrade both methods fail.

**Comment by atymic** [comment]
Thanks for the detailed report. I think we might just revert the PR for the moment.

Could you provide a stack trace for them both if possible please?

cc @reliq

**Comment by atymic** [comment]
Okay, first issue is fixed. Could you post the params you are using for the second one please?

**Comment by atymic** [comment]
Ahh, I misunderstood. The first one is currently broken in 3.0, I see that it had some hardcoded config access. I'll make a ticket for that. 

For the second one, found the issue and fix incoming.

**Comment by atymic** [comment]
Okay, tested with:

```
        $tweets = Twitter::directQuery(
            'https://publish.twitter.com/oembed',
            'GET',
            ['url' => 'https://twitter.com/jbrooksuk/status/1351207981141127171']
        );
``` 

returns 

```
{#1331 ▼
  +"url": "https://twitter.com/jbrooksuk/status/1351207981141127171"
  +"author_name": "James Brooks"
  +"author_url": "https://twitter.com/jbrooksuk"
  +"html": """
    <blockquote class="twitter-tweet"><p lang="en" dir="ltr">I just finished deploying the first version of my <a href="https://twitter.com/laravelphp?ref_src=twsrc ▶
    <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
    """
  +"width": 550
  +"height": null
  +"type": "rich"
  +"cache_age": "3153600000"
  +"provider_name": "Twitter"
  +"provider_url": "https://twitter.com"
  +"version": "1.0"
}
```

Fix tagged in 3.1.2, i've opened #344 for the other issue which we will fix :)
